### DISCUSSION TITLE
SQL上线配置问题

### DISCUSSION THREAD

**Discussion by dmz-leon** [hhyo/Archery#1899] [Q&A]
想要在SQL上线的时候，将检查等级为warning的语句给停止继续执行，应该修改哪个配置？

**Comment by nick2wang** [comment]
auto_review_wrong设置为1即可
![image](https://user-images.githubusercontent.com/33473924/194735886-3fb949e0-d8a1-449e-ba27-741e4f767f4c.png)

**Reply by dmz-leon** [reply]
3Q
